### ISSUE TITLE
Disable fee levels when sending strat

### ISSUE THREAD

**Issue by mikedennis** [stratisproject/Breeze#348]
Since there is only one fee level for strat we could disable these options except for bitcoin to avoid confusion. I've seen this reported a couple of different times in the community testing with the beta.

**Comment by dev0tion** [comment]
Yeah I agree, people get confused because it's the same fee for all types of transactions.
What do you think @bokobza?

**Comment by bep42** [comment]
We just have to display it without the selection option :)
